app/code/community/Ebizmarts/MailChimp/Model/Api/Customers.php

```diff
@@ -88,6 +88,7 @@ public function createBatchJson($mailchimpStoreId, $magentoStoreId)
     {
```

Comment:
Function's nesting level (6) exceeds 5; consider refactoring the function
https://travis-ci.org/mailchimp/mc-magento/jobs/564988692#L832